components/src/core/drawer/drawer-layout/drawer-layout.component.scss

```diff
@@ -9,7 +9,7 @@ $transition: 300ms cubic-bezier(0.4, 0, 0.2, 1);
 [dir='rtl'] .pxb-drawer-layout-content .mat-drawer-side {
     border-left: 0; // Border-left is provided by the Drawer component.
     .pxb-drawer-layout-sidenav.pxb-drawer-layout-shadow {
-        box-shadow: 0 0 -4px 0 #424e54;
+        box-shadow: 0 0 -0.25rem 0 #424e54;
```

Comment:
I don't think we will need this for the box-shadow... Would be better if this logic is moved to the component itself, and do something like `[class.mat-elevation-z6]="!sideBorder"`

Reply:
the purpose of a box shadow is to add elevation to a surface. I don't think people needing extra large font size also need an "extra deep" UI

Reply:
This is a good idea; I think this was done on the Drawer component but not on the Drawer Layout.   Good catch.